docs/sources/dashboards/build-dashboards/annotate-visualizations/index.md

```diff
@@ -78,6 +78,22 @@ In Grafana v5.3+ it's possible to use template variables in the tag query. So if
 
 {{< figure src="/static/img/docs/annotations/annotation_tag_filter_variable-8-1-0.png" max-width="600px" >}}
 
+### Filter by panel
+
+You can configure to which panels to apply the annotations by choosing an option from the _Show in_ section. You can apply the annotations to `All panels` that support annotations, or filter specific panels by choosing `Selected panels` or `All panels except`. The annotations will be displayed accordingly.
+
+{{< figure src="/media/docs/grafana/screenshot-grafana-10-0-annotation-filtering.png" max-width="600px" caption="Annotation filtering" >}}
+
+### Add time region
+
+When adding or editing an annotation, you can define a repeating time region by setting _Query type_ to `Time regions` - simply define the _From_/_To_ sections with the preferred days of the week and time. You also have the possibility to change the timezone - the default being set to dashboard's timezone.
```

Comment:
Should we maybe mention that the datasource is -- Grafana -- here?

Reply:
I think annotations are only available with grafana and gdev-testdata, so I would expect not to be that hard to find. But I might be wrong..
Maybe `Note: Time region query type is available for -- Grafana -- datasource`?

Reply:
good point, also the example image already shows this info too so probably ok to leave out 👍